consensus/types/src/execution_payload_header.rs

```diff
@@ -164,6 +167,30 @@ impl<T: EthSpec> ExecutionPayloadHeaderCapella<T> {
     }
 }
 
+impl<T: EthSpec> ExecutionPayloadHeaderDeneb<T> {
+    pub fn upgrade_to_electra(&self) -> ExecutionPayloadHeaderElectra<T> {
+        ExecutionPayloadHeaderElectra {
+            parent_hash: self.parent_hash,
+            fee_recipient: self.fee_recipient,
+            state_root: self.state_root,
+            receipts_root: self.receipts_root,
+            logs_bloom: self.logs_bloom.clone(),
+            prev_randao: self.prev_randao,
+            block_number: self.block_number,
+            gas_limit: self.gas_limit,
+            gas_used: self.gas_used,
+            timestamp: self.timestamp,
+            extra_data: self.extra_data.clone(),
+            base_fee_per_gas: self.base_fee_per_gas,
+            block_hash: self.block_hash,
+            transactions_root: self.transactions_root,
+            withdrawals_root: self.withdrawals_root,
+            blob_gas_used: 0,
+            excess_blob_gas: 0,
```

Comment:
Should carry this values from deneb payload

Reply:
Yep good catch